### ISSUE TITLE
How can you set the titlebar background color to a dynamic resource?

### ISSUE THREAD

**Issue by crclayton** [MahApps/MahApps.Metro#1966]
I know I can change the titlebar color if I change the accent, but I want the titlebar to stay the same if I change the accent, and only change if I change the base theme. 

I know I can do this:

     <MetroWindow TitleForeground="{DynamicResource WhiteBrush}">

But I can't do this:

     <MetroWindow TitleBackground="{DynamicResource WhiteBrush}">

So instead I tried this: 

     <MetroWindow Style="{DynamicResource TitleBackground}">

    <Window.Resources>

        <Style x:Key="TitleBackground" TargetType="{x:Type controls:MetroWindow}">
            <Setter Property="Background" Value="{DynamicResource WhiteBrush}" />
        </Style>

But the accent I'm using seems to override that.

I also know I can use clean style as outlined here to not let the accent change the titlebar color: https://github.com/MahApps/MahApps.Metro/issues/1025

However, clean style comes with other changes I'm not interested in, and also still doesn't let me choose custom dynamic resources. 

So in short, how can I accomplish this: 

     <MetroWindow TitleBackground="{DynamicResource WhiteBrush}">

Thanks for the help.

**Comment by crclayton** [comment]
As punker76 explained later in the linked issue, what I was looking for was: 

     <MetroWindow WindowTitleBrush="{DynamicResource WhiteBrush}">
